pydiscourse/client.py

```diff
@@ -1376,6 +1381,11 @@ def _request(self, verb, path, params={}, files={}, data={}, json={}):
                 timeout=self.timeout,
             )
 
+            overridden_kwargs = default_kwargs.copy()
+            overridden_kwargs.update(request_kwargs)
+
+            response = requests.request(verb, url, **overridden_kwargs)
```

Comment:
I think this is fine (NO CHANGE NEEDED) but you could simplify this by just updating the `default_kwargs` with the `request_kwargs` and passing through the `default_kwargs`. It's not material that the `default_kwargs` are modified since they're recreated in the method scope. Again, no change needed!

Reply:
Cleaned that up, and it made sense to adjust the naming of arguments in the process.